lib/Tools/LinearLayout.cpp

```diff
@@ -1,6 +1,7 @@
 #include "triton/Tools/LinearLayout.h"
 
 #include <cstdint>
+#include <set>
```

Comment:
nit, is std::set used in here?

Reply:
Initially I thought so, but I found it's used in `getFreeVariableMasks`